roadmap.md

```diff
@@ -1,3 +1,5 @@
+# Roadmap
+
 **Note that the expected quarter when a feature will be completed based on very rough estimates of the effort involved and therefore can change. Items beyond CQ4'22 will be planned in detail and confirmed at the start of the quarter depending on progress made in previous quarter and available bandwidth.**
```

Comment:
I'm not sure exactly what the second sentence is saying. It will also become out of date quite quickly... I think maybe something more general like "Items to be delivered beyond the current quarter may move, or change in scope".

Reply:
Done but changed "may move" to "may be pushed back" as clearer.